dxr/static_unhashed/js/code-highlighter.js

```diff
@@ -132,6 +138,26 @@ $(function () {
         permalink.attr('href', permalink_href + windowHash);
     }
 
+    //replace any occurrence of {{line}} in hrefs with the last-selected line.
+    //unless the last-selected line is undefined, then remove {{line}} from the
+    //displayed url.
+    function updateNavLinks(lastNumber) {
+        navlinks.each(function() {
+            var $this = $(this);
+            // Copy the href to a template attr, since we will be updating it
+            // and don't want to lose the template. This behavior will happen
+            // only the first time this function is called.
+            if (!$this.data('template')) {
+                $this.data('template', $this.attr('href'));
+            }
+            if (lastNumber) {
+                $this.attr('href', $this.data('template').replace(/{{line}}/g, lastNumber));
```

Comment:
It did work when I tested it locally. I think curlies are only used in {n} to match n occurrences of the preceding character.